src/models/instance.rs

```diff
@@ -295,9 +295,11 @@ impl Default for ForegroundEffect {
 #[serde(default, rename_all = "camelCase", deny_unknown_fields)]
 pub struct InstanceCapture {
     pub system_enable: bool,
+    pub system_grabber_device: String,
```

Comment:
Looking at the original source, it seems they're storing "NONE" in the config to signify there is no value. I think we could keep this saner by using a `Option<String>` with a `#[serde(deserialize_with = "...")]` attribute?

Reply:
Would something like that be alright?

```rs
// src/serde/config_option.rs
use std::fmt;

/// Serde visitor for deserializing string values which might
/// be literally a "NONE"-string
struct ConfigOptionalVisitor;

impl<'a> serde::de::Visitor<'a> for ConfigOptionalVisitor {
    type Value = Option<String>;

    fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
        formatter.write_str("A useful type-string or \"NONE\"-string")
    }

    fn visit_str<A>(self, string: &str) -> Result<Self::Value, A>
    where
        A: serde::de::Error,
    {
        match string.to_lowercase() {
            "NONE" => None,
            _ => Some(string.to_owned()),
        }
    }
}

/// Decode a ConfigOptional value
///
/// # Parameters
///
/// `deserializer`: Serde deserializer
pub fn from_config_optional<'de, D>(
    deserializer: D,
) -> std::result::Result<Option<String>, D::Error>
where
    D: serde::Deserializer<'de>,
{
    deserializer.deserialize_str(ConfigOptionalVisitor {})
}
```